libc-bottom-half/cloudlibc/src/libc/sys/socket/accept.c

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: BSD-2-Clause
+
+#include <common/errno.h>
```

Comment:
In cloudlibc, this header handles translating from cloudabi error codes such as `ENOTCAPABLE` to POSIX error codes, but in wasi-libc, we decided to just have the WASI calls return the desired error codes directly, so this header and `errno_fixup_socket` doesn't do anything. I think if we removed this header, it would be straightforward to move this file into libc-bottom-half/sources.

Reply:
done